README.md

```diff
@@ -373,16 +375,20 @@ will give a more in-depth description of the command and available options.
 #### Proxy Submissions
 
 The autograder allows course staff to submit on behalf of students using "proxy" submissions.
-The proxy submit command adds two parameters to the normal submission command:
+The suite of proxy commands use the following two new concepts:
 
- - `proxy-email` -- The student that the course staff is submitting on behalf of.
- - `proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
+ - `--proxy-email` -- The student that the course staff is submitting on behalf of.
+ - `--proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
    - If you want to set the time manually, time is the number of milliseconds from the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).
+   We can use a [Unix Epoch time converter](https://www.epochconverter.com/) to help with the conversion.
```

Comment:
s/We/You

Either is fine in this document (since we are not too strict),
but you just want to be consistent with your section.
(The previous sentence uses "You".)